README.md

```diff
@@ -665,6 +667,17 @@ Besides libraries, there're [Collection on Codepen](http://codepen.io/collection
 
 # Worth Reading
 
+* Java
+  * [Influential Books](https://github.com/akullpp/awesome-java#influential-books)
+  * [Awesome Java](https://github.com/akullpp/awesome-java#websites)
+
+* PHP
```

Comment:
sorry, because there is still much interaction between the two frameworks. But I understand that it is only javascript :D